README.md

```diff
@@ -40,81 +189,133 @@ The following is a non-normative example of how an RP would use the `claims` par
     &client_id=https%3A%2F%2Fclient.example.org%2Fcb
     &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb
     &scope=openid
-    &claims=...
+    &claims=claims=%7B%22id_token%22%3A%7B%22vc%22%3A%7B%22types%22%3A%5B%22https%3A%2F%
+     2Fdid.itsourweb.org%3A3000%2Fsmart-credential%2FOntario-Health-Insurance-Plan
+     %22%5D%7D%7D%7D
     &state=af0ifjsldkj
-    &nonce=n-0S6_WzA2Mj
+    &nonce=960848874
     &registration_uri=https%3A%2F%2F
       client.example.org%2Frf.txt%22%7D
       
 ```
-### claims parameter (simple)
+#### claims parameter
 
-In its simplest form, the RP just asks the OP to provide a VP or VC containing a set of claims in the response by listing those claims underneath `vp_token`.  
+In this case, the RP asks the OP to provide a VC of a certain type.  
 
 ```
 {
     "id_token": {
-        "acr": null
-    },
-    "vp_token": {
-        "given_name": null,
-        "family_name": null,
-        "birthdate": null
+      "vp": {
+        "credential_types": ["https://did.itsourweb.org:3000/smartcredential/Ontario-Health-Insurance-Plan"]
+      } 
     }
 }
 ```
 
-### claims parameter (with identity assurance requirements)
-
-The RP may also request claims fulfilling the requirtements of a certain trust framework and identity assurance level. It does so by adding a `verified_claims` claims element to the `vp_token` section, which defines the verification requirements and lists the respective End-User claims. 
-Note: this syntax was adopted from the OpenID Connect for Identity Assurance spec (https://openid.net/specs/openid-connect-4-identity-assurance-1_0.html).
+### Authentication Response 
 
-```
-{
-    "id_token": {
-        "acr": null
-    },
-    "vp_token": {
-        "verified_claims": {
-            "verification": {
-                "trust_framework": {
-                    "value": "eidas"
-                },
-                "identity_assurance_level": {
-                    "value": "high"
-                }
-            },
-            "claims": {
-                "given_name": null,
-                "family_name": null,
-                "birthdate": null
-            }
-        }
-    }
-}
-```
-## Authentication Response 
-
-The successful authentication response contains a `vp_token` parameter along with  `id_token` and `state`.
+The successful authentication response contains an `id_token` and `state`.
 ```
   HTTP/1.1 302 Found
   Location: https://client.example.org/cb#
-    id_token=eyJ0 ... NiJ9.eyJ1c ... I6IjIifX0.DeWt4Qu ... ZXso
-    &vp_token=...
+    id_token=...
     &state=af0ifjsldkj
       
 ```
+### Verifiable Presentation
+
+The ID Token contains a `vp` element with the Verifiable Credential data. 
+
+```json
+{
+   "iss":"https://book.itsourweb.org:3000/wallet/wallet.html",
```

Comment:
@AdamJLemmon is this supposed to by a SIOP compliant id token? I'm asking since although it contains a "sub_jwk! the "iss" claim is not "https://self-isued.me". How is the RP supposed to validate the signature of this id token?

Reply:
We have been advocating for expanding the `iss` claim for self signed tokens to not be constrained to self-issued.me.
Here that claim is the URL of the wallet which is returning the `id_token`.  
Our rationale was that the RP would detect the presence of `sub_jwk` and know the `id_token` was self signed instead of checking that the `iss` claim is self-issued. So if sub_jwk is present then validate the signature against that public key otherwise follow existing protocol.  

We could update this to be self-issued.me though if we wanted to present this as being compliant with OIDC CORE as it sits today?

Reply:
I understand your rationale and sympathies with them. self-issued.me is much too limited. I would rather prefer an issuer specific iss value and an explicit marker for self issued id tokens. The presence of the sub_jwk could serve as such a marker. 

However, I don't think this example is OIDC Core compliant. According to the OIDC Core https://openid.net/specs/openid-connect-core-1_0.html#IDTokenValidation the client shall use the keys provided by the issuer to check the signature.

Reply:
Right, so your suggestion for the time being in this doc would be to revert `iss` to self-issued.me for this example?